pupil_src/shared_modules/raw_data_exporter.py

```diff
@@ -105,12 +105,24 @@ class Raw_Data_Exporter(Analysis_Plugin_Base):
         '''
     icon_chr = chr(0xe873)
     icon_font = 'pupil_icons'
+    
+    def __init__(self, g_pool, export_pupil_positions=True, export_pupil_gaze_positions_info=True, export_gaze_positions=True):
+        super().__init__(g_pool)
+        self.export_pupil_positions = export_pupil_positions
+        self.export_pupil_gaze_positions_info = export_pupil_gaze_positions_info
+        self.export_gaze_positions = export_gaze_positions
```

Comment:
Would it be possible to rename these to `should_export_*`? Also, may I suggest to rename `export_pupil_gaze_positions_info` to `should_export_field_info`?